salt/runners/nacl.py

```diff
@@ -411,12 +420,13 @@ def sealedbox_encrypt(data, **kwargs):
     .. code-block:: bash
 
         salt-run nacl.sealedbox_encrypt datatoenc
-        salt-call --local nacl.sealedbox_encrypt datatoenc pk_file=/etc/salt/pki/master/nacl.pub
-        salt-call --local nacl.sealedbox_encrypt datatoenc pk='vrwQF7cNiNAVQVAiS3bvcbJUnF0cN6fU9YTZD9mBfzQ='
     '''
+    # ensure data is bytes
+    data = salt.utils.stringutils.to_bytes(data)
+
     pk = _get_pk(**kwargs)
     b = libnacl.sealed.SealedBox(pk)
-    return base64.b64encode(b.encrypt(data))
+    return base64.b64encode(b.encrypt(salt.utils.stringutils.to_bytes(data)))
```

Comment:
Sorry to nitpick at this, while applying the changes to locally I noticed this line.
I don't think this one is present in salt.modules.nacl yet the function there is the same as here!